src/Libraries/CoreNodeModels/ColorPalette.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Xml;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows.Media;
+using ProtoCore.AST.AssociativeAST;
+using DSColor = DSCore.Color;
+using Autodesk.DesignScript.Runtime;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph;
+
+using CoreNodeModels.Properties;
+
+namespace ColorNode
+{
+    /// <summary>
+    ///     Custom Color UI node.
+    /// </summary>
+    [NodeName("Color Picker")]
```

Comment:
This file should be deleted and is not in my code, it must've been included when i first setting up sorry.